cfme/tests/webui/test_advanced_search.py

```diff
@@ -179,14 +91,8 @@ def test_can_open_advanced_search(param, appliance):
     view.search.close_advanced_search()
 
 
-@pytest.mark.uncollectif(
-    lambda param, appliance: (
-        param.collection in (ConfigManager, 'ansible_tower_providers') or
-        param.filter == 'Job Template (Ansible Tower) : Name'
-    )
-)
 @pytest.mark.meta(automates=[BZ(1402392)])
```

Comment:
@pavelzag I'm curious how this metadata parsed?

Reply:
I think this comment was updated, and I'm no so sure the update triggers a notification. @psimovec I'm also interested in how this meta gets applied.

Reply:
@john-dupuy @psimovec Using `miq bz report cfme/tests/webui/test_advanced_search.py` it looks like this meta marker on the base function here does not carry through that function being methodized and decorated.

```
 ERROR: File bz-report.yaml not found, something went wrong during report generation.
       Likely no BZs were found in cfme/tests/webui/test_advanced_search.py with 'automates'/'coverage', so a report wasn't generated.```

Reply:
@john-dupuy @mshriver unfortunately it doesn't.. I have tried to fix that, but failed.. it will need bigger changes to be able to do that. For now I have added this mark to module level pytestmark, so it is visible from outside.. And I should fix it in another PR